database/migrations/0044-adapt_users_table_for_gradidoid.ts

```diff
@@ -0,0 +1,42 @@
+/* MIGRATION TO ADD GRADIDO_ID
+ *
+ * This migration adds new columns to the table `users` and creates the
+ * new table `user_contacts`
+ */
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export async function upgrade(queryFn: (query: string, values?: any[]) => Promise<Array<any>>) {
+  await queryFn(`
+      CREATE TABLE IF NOT EXISTS \`user_contacts\` (
+        \`id\` int(10) unsigned NOT NULL AUTO_INCREMENT,
+        \`type\` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL,
+        \`users_id\` int(10) unsigned NOT NULL,
+        \`email\` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL UNIQUE,
+        \`email_hash\` binary(32) NULL,
+        \`email_checked\` tinyint(4) NOT NULL DEFAULT 0,
+        \`phone\` varchar(255) COLLATE utf8mb4_unicode_ci NULL DEFAULT NULL,
+        \`created_at\` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+        \`updated_at\` datetime NULL DEFAULT NULL,
+        \`deleted_at\` datetime NULL DEFAULT NULL,
+        PRIMARY KEY (\`id\`)
+      ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;`)
+
+  await queryFn('ALTER TABLE `users` ADD COLUMN `gradido_id` varchar(36) NULL AFTER `id`;')
```

Comment:
```suggestion
  await queryFn('ALTER TABLE `users` ADD COLUMN `gradido_id` UUID NOT NULL DEFAULT UUID() AFTER `id`;')
```

Reply:
I did not try it. But if this works, we already have the gradidio ID for all users.